forge-gui/res/cardsfolder/h/harrowing_swarm.txt

```diff
@@ -0,0 +1,10 @@
+Name:Harrowing Swarm
+ManaCost:1 G
+Types:Sorcery
+A:SP$ ManifestDread | SubAbility$ DBAnimate | SpellDescription$ Manifest dread.
+SVar:DBAnimate:DB$ AnimateAll | ValidCards$ Creature.faceDown+YouCtrl | Zone$ Battlefield | staticAbilities$ ReduceCost | Triggers$ TurnedFaceUp | Duration$ Permanent | StackDescription$ SpellDescription | SpellDescription$ Then each face-down creature you control gains "This permanent costs {2} less to turn face up" and "When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it."
+SVar:ReduceCost:Mode$ ReduceCost | Amount$ 2 | Type$ TurnFaceUp | ValidCard$ Card.Self | Activator$ You | EffectZone$ All | Description$ This permanent costs {2} less to turn face up.
+SVar:TurnedFaceUp:Mode$ TurnFaceUp | ValidCard$ Creature.Self | Execute$ TrigPump | TriggerZones$ Battlefield | TriggerDescription$ When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it. | Description$ When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it.
```

Comment:
intervening if not handled :p

Reply:
Wouldn't this be covered by `ValidCard$ Creature.Self` ?

Reply:
no, that's only checked for the trigger event